NewForce-Cohort-4/tabloidmvc-team-neurotic-ninjas
# Create tag

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change
Please delete options that are not relevant.

# Testing Instructions
- User should login in with email admin@admin.com 
- User should go to the Tag Management Menu Link
- User should create a tag and save
- New Tag Should appear
